_pkgdown.yml

```diff
@@ -1,20 +1,20 @@
 url: https://multilevelmod.tidymodels.org
 
 template:
+  bootstrap: 5
+  bslib:
+    danger: "#CA225E"
+    primary: "#CA225E"
   package: tidytemplate
-  params:
-    theme: tidymodels
-    part_of: <a href="https://github.com/tidymodels">tidymodels</a>
-    footer: multilevelmod is a part of the <strong>tidymodels</strong> ecosystem, a collection of modeling packages designed with common APIs and a shared philosophy.
-
-# https://github.com/tidyverse/tidytemplate for css
 
 development:
   mode: auto
 
-
 figures:
   fig.width: 8
   fig.height: 5.75
 
-
+navbar:
+  left:
+  - text: Get Started
+    href: articles/Get_Started.html
```

Comment:
I don't believe you need to or can do it like this. As long as it is in a folder, it will be discovered.